classes/Validate.php

```diff
@@ -751,11 +751,11 @@ public static function isZipCodeFormat($zip_code)
      *
      * @param string $way Keyword to validate
      *
-     * @return int Validity is ok or not
+     * @return bool Validity is ok or not
      */
     public static function isOrderWay($way)
     {
-        return $way === 'ASC' | $way === 'DESC' | $way === 'asc' | $way === 'desc';
+        return !empty($way) && in_array(strtoupper($way), ['ASC', 'DESC']);
```

Comment:
I'm not sure you preserved the behavior. This is bitwise here it will return an int, not a bool?